lang/cs/Org.Apache.REEF.Bridge.Client/Org.Apache.REEF.Bridge.Client.DotNet.csproj

```diff
@@ -0,0 +1,57 @@
+<Project>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+  <PropertyGroup>
+    <AssemblyName>Org.Apache.REEF.Bridge.Client</AssemblyName>
+    <Description>Bridge CSharp Client for REEF.NET</Description>
+    <PackageTags>REEF Bridge CSharp Client</PackageTags>
+  </PropertyGroup>
+  <Import Project="..\build.DotNetApp.props" />
+  <ItemGroup>
+    <PackageReference Include="Grpc" Version="1.11.0-pre2" />
+    <PackageReference Include="Grpc.Tools" Version="1.11.0-pre2" />
+  </ItemGroup>
+  <PropertyGroup>
+    <GrpcSchemaDirectory>..\..\common\proto\bridge</GrpcSchemaDirectory>
+    <GrpcToolsDirectory>$(SolutionDir)\packages\grpc.tools\1.11.0-pre2\tools\windows_x64</GrpcToolsDirectory>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net452|x64'">
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <WarningsAsErrors />
+  </PropertyGroup>
+  <!-- TODO[JIRA REEF-1888] This item group will not be needed when only .netcore is targeted. -->
+  <ItemGroup Condition="'$(TargetFramework)' == 'net452' Or '$(TargetFramework)' == 'net46'">
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Configuration" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Org.Apache.REEF.Tang\Org.Apache.REEF.Tang.DotNet.csproj" />
+    <ProjectReference Include="..\Org.Apache.REEF.Utilities\Org.Apache.REEF.Utilities.DotNet.csproj" />
+    <ProjectReference Include="..\Org.Apache.REEF.Common\Org.Apache.REEF.Common.DotNet.csproj" />
+    <ProjectReference Include="..\Org.Apache.REEF.Driver\Org.Apache.REEF.Driver.DotNet.csproj" />
+    <ProjectReference Include="..\Org.Apache.REEF.Wake\Org.Apache.REEF.Wake.DotNet.csproj" />
+  </ItemGroup>
+  <Import Project="..\build.DotNet.targets" />
```

Comment:
Line 25 imports the one for apps. Why do we need this as well?

Reply:
The new project follows Scott's model so that as he continues to evolve the projects moving to Linux, the new bridge client project is exactly the same in structure as the current evaluator DotNet version of the project.

Reply:
To give some context - any project that generates an executable should pull in the DotNetApp.props file as this specified netcoreapp2.0 as the targeted framework.